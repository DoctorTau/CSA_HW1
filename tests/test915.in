264
8349 2063 -6136 -1546 -7461 6084 -8201 6801 3621 2097 -3861 -2244 8401 6717 3648 5479 6958 -1200 -4449 -421 4879 -9256 -2007 -3189 4369 4497 9526 -6573 627 3523 7656 -3986 -4342 890 -6782 5740 49 4082 4889 -5124 758 108 3177 717 2970 3635 -5913 3215 9026 -7995 -5817 -859 793 8246 5400 7593 -1496 7700 -4447 -6711 91 -1752 8178 -9991 2819 -7536 1429 -5875 -4940 8736 -4446 -6559 -1053 -426 -359 -2684 -1473 7653 2823 8943 5820 -1915 2010 4789 3302 -1763 -6878 -932 -194 6062 -9690 -1363 9970 -389 -1841 7704 -5613 3361 -1839 4250 -5681 7520 -7470 -3816 -170 828 2764 -6137 569 8229 736 9663 4994 8109 -4270 -5003 -4959 -429 -6237 7250 3975 495 4061 7674 -3687 -1759 2491 8930 -1146 -6090 -5402 7234 7398 6745 -4780 3240 -1949 194 -2255 8479 1794 7343 5501 8217 -8341 4051 -3167 -5439 8162 7170 3981 -9205 -3298 2135 -5901 9988 -8258 3738 5313 -9950 -1942 6498 8132 -3232 -1307 5966 215 -6556 7482 9299 5477 4342 300 -2649 5292 4867 3902 -3874 -9114 -4160 -1415 7639 1204 -761 -8773 9420 -3592 -5084 -8106 6817 5273 6004 -6277 6120 3870 2937 5067 -7273 -3305 1252 6015 9679 -8210 -9197 5410 5689 -6944 6154 2516 8084 2244 -3805 -9684 -4623 -992 -3861 -7573 -2042 -1821 -9628 -467 6079 4616 -9959 792 7309 -9858 946 5399 5100 455 -2589 9806 -789 -7061 8890 -3929 9353 7888 1865 -974 -8328 232 -6487 8436 9743 -7632 -8324 6928 1855 7147 9545 -375 -9443 9138 -6266 9199 3293 -8814 2376 -5230 -5717 -3759 -3394 