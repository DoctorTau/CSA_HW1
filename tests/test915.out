-6136 -1546 -2244 -1200 -9256 -3986 -4342 -6782 -5124 -1496 -1752 -7536 -4940 -4446 -426 -2684 -6878 -932 -194 -9690 -7470 -3816 -170 -4270 -1146 -6090 -5402 -4780 -3298 -8258 -9950 -1942 -3232 -6556 -3874 -9114 -4160 -3592 -5084 -8106 -8210 -6944 -9684 -992 -2042 -9628 -9858 -974 -8328 -7632 -8324 -6266 -8814 -5230 -3394 